.storybook/assets/css/preview.css

```diff
@@ -42,6 +50,68 @@
   max-width: calc(var(--base-unit) * 48);
 }
 
-:global(.sbdocs.sbdocs-table tr:nth-of-type(2n)) {
-  background-color: transparent;
+/* Markup Tables on Docs pages */
+:global(.sb-unstyled thead) {
+  background-color: var(--color-surface--background);
+}
+
+:global(.sb-unstyled table tr th),
+:global(.docblock-argstable-head tr th) {
+  color: var(--color-text) !important;
+}
+
+:global(.sb-unstyled table tr td) {
+  background-color: var(--color-surface--background--subtle);
+  border: solid 1px var(--color-border);
+  color: var(--color-text);
+}
+
+/* <Canvas/> & similar code blocks on Docs pages */
```

Comment:
![Screenshot 2024-11-21 at 1 27 43 PM](https://github.com/user-attachments/assets/e8f80a11-cbc6-4321-89f2-7c3d68243b28)
![Screenshot 2024-11-21 at 1 27 50 PM](https://github.com/user-attachments/assets/e8f98cad-29aa-4e0e-b8be-df17f88468ca)

Reply:
@chris-at-jobber I saw on the new docs page you have set colors like yellow, pink etc. I only noticed that after I chose the colors here. 

My justification for the colors was to match the code sandbox.  I think I'd be good to match your color scheme in new docs but that might mean readjusting the background color

![Screenshot 2024-11-21 at 1 33 14 PM](https://github.com/user-attachments/assets/c3890533-cd7f-4418-9b37-9def27c2b83c)

![Screenshot 2024-11-21 at 1 33 04 PM](https://github.com/user-attachments/assets/6f75b062-fab3-498c-bc4c-56a51f231c6b)